src/components/sbb-card/readme.md

```diff
@@ -1,49 +1,78 @@
-The `sbb-card` component is a generic content container as an action element; its task is to contain HTML elements related to a single subject.
+The `sbb-card` component is a generic content container; its task is to contain content related to a single subject. 
+There are various sizes (affecting paddings) and colors available.
 
-Internally, it is either rendered as an anchor (`<a>`, when used with a `href` attribute) or as a button (when used without a `href` attribute).
-When rendered as a button, it is possible to listen to the click event to add behavior.
+The `sbb-card-badge` component can be used via slot to display a badge in the upper right corner.
```

Comment:
`via slot` can maybe be removed or rephrased, as the slotting is done internally and is therefore just an implementation detail the consumer does not need to know.